src/test/java/com/uid2/admin/vertx/ClientKeyServiceTest.java

```diff
@@ -235,6 +235,45 @@ public void clientRolesCreatesSiteKeyIfNoneExists(Set<Role> roles, boolean siteK
         });
     }
 
+    @Test
+    public void listBySiteStringId(Vertx vertx, VertxTestContext testContext){
+        fakeAuth(Role.CLIENTKEY_ISSUER);
+
+        get(vertx, testContext, "api/client/listBySite/test", expectHttpStatus(testContext, 400));
+    }
+
+    @Test
+    public void listBySiteInvalidId(Vertx vertx, VertxTestContext testContext){
+        fakeAuth(Role.CLIENTKEY_ISSUER);
+
+        get(vertx, testContext, "api/client/listBySite/0", expectHttpStatus(testContext, 400));
+    }
+
+    @Test
+    public void listBySiteUnusedId(Vertx vertx, VertxTestContext testContext){
+        fakeAuth(Role.CLIENTKEY_ISSUER);
+
+        get(vertx, testContext, "api/client/listBySite/100", expectHttpStatus(testContext, 404));
+    }
+
+    @Test
+    public void listBySiteWorking(Vertx vertx, VertxTestContext testContext) {
```

Comment:
I know there's some questions about naming conventions for our tests, but I think we can drop the `Working` at least: `listBySite`.

Reply:
Okay cool. Will do that.